test/integration/auth/svcaccttoken_test.go

```diff
@@ -205,7 +205,7 @@ func TestServiceAccountTokenCreate(t *testing.T) {
 		}
 
 		checkPayload(t, treq.Status.Token, `"system:serviceaccount:myns:test-svcacct"`, "sub")
-		checkPayload(t, treq.Status.Token, `["api"]`, "aud")
+		checkPayload(t, treq.Status.Token, `"api"`, "aud")
```

Comment:
this change is surprising... does the serialization actually modify the shape of the service account tokens we emit? (string `aud` field instead of array of strings)?

Reply:
This was introduced by https://github.com/square/go-jose/pull/247 — tokens with a single audience are emitted with a string field rather than an array of strings.